src/stylesheets/main.scss

```diff
@@ -170,7 +172,8 @@ body {
   $copy-button-colour: #00823b;
 
   @include govuk-font(16);
-  @include govuk-focusable;
+  // TODO: Need to make copy button focusable/hoverable but we need some design decision to be made
```

Comment:
Good TODOs, maybe we should raise issues to make sure we dont forget about these?

Reply:
Good suggestion, I'll add it to https://github.com/alphagov/design-system-team-internal/issues/106